src/RF24LogAppender.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * RF24LogAppender.h
+ *
+ *  Created on: 2 paź 2020
+ *      Author: wmarkowski
+ */

```

Comment:
This is a place where the license header available at the bottom of LICENSE.txt should be used

Reply:
OK.

Reply:
This was already well handled by @2bndy5 but I see that the examples sources (under **examples** directory) have a different header format that the sources (under **src**) folder. @2bndy5, is there any specific reason behind? The licens headers from **src** look nicer. Can I just copy them to the **examples** sources?

I will translate also the date of file creation from Polish to English.

Reply:
It's probably more legally binding of we use the copyright notice that I added to the src files. The examples currently direct readers to the license in lib root, but that may be incorrect/improper (depends on what the license says).

Reply:
Ok, so I will use the copyright notice from src files.

Reply:
> The examples currently direct readers to the license in lib root, but that may be incorrect/improper (depends on what the license says).

It should be fine, but a license header is a bit more explicit.